### ISSUE TITLE
Cannot Find Package (Same as #1128)

### ISSUE THREAD

**Issue by QwertyQaz414** [go-redis/redis#1174]
**Pls, don't close the issue if it hasn't been solved yet :)**

I'm not a docker expert so please, if this is just a config error or something that I did wrong, create a piece of documentation of how to use it properly inside docker. Thanks!

# Steps
1.  create simple main.go file that uses redis
2. create dockerfile:
```dockerfile
FROM golang:1.8.7-alpine3.6
RUN mkdir /app
WORKDIR /app
COPY main.go /app

# get git
RUN apk update
RUN apk add git

# This here breaks
RUN go get -u github.com/go-redis/redis
RUN go build -o main .
CMD ["./main"]
```
3. build (I used a docker-compose .yml and `docker-compose build <name>`)

# Error

![image](https://user-images.githubusercontent.com/11825235/66325109-faf68600-e926-11e9-8e96-fe020a14eb9b.png)

**Comment by vmihailenco** [comment]
You need to use [Go modules](https://github.com/golang/go/wiki/Modules) that are available from v1.11 and `import github.com/go-redis/redis/v7`.

**Comment by QwertyQaz414** [comment]
For anyone having the same problems (and cannot switch to go modules) put this in your dockerfile...

```dockerfile
FROM golang:1.8.7-alpine3.6
RUN apk update
RUN apk add git

WORKDIR /go/src/github.com/go-redis/redis/
RUN git clone https://github.com/go-redis/redis.git .
RUN git checkout v6

WORKDIR /app
COPY main.go /app

RUN go build main.go

CMD ["./main"]
```
